src/classes/ADDR_SmartyStreets_Validator.cls

```diff
@@ -143,69 +143,70 @@ public with sharing class ADDR_SmartyStreets_Validator implements ADDR_IValidato
 
         //Don't even make the address verification request to SmartyStreets if no US address is included, to save in API calls
         if(includesUSAddress) {
```

Comment:
If a US Address isn't included, it doesn't set verified on the address. It could be verified and set as a foreign address.

Reply:
Good question. This is feels like a spec issue that we can discuss with @mpusto tomorrow.

Reply:
@mpusto Randi noticed that our SmartyStreets address verification code filters out non-US addresses. It seems that when we implemented the feature years ago, they only supported US address verification, but in visiting smartystreets.com it seems like they now support international addresses. I think we could use a work item to investigate updating our code to stop filtering out international addresses with SmartyStreets, what do you think?

Reply:
Yes, I agree! It's come up in the HUB before and I answered with "well they didn't support it when we wrote the code". :)

Reply:
Perfect, I'll let you add it to the backlog! Thanks.

Reply:
https://github.com/SalesforceFoundation/Cumulus/issues/2853